library/manageiq_user.py

```diff
@@ -0,0 +1,268 @@
+#!/usr/bin/python
+
+
+DOCUMENTATION = '''
+---
+module: manageiq_user
+short_description: management of users in ManageIQ
+requirements: [ ManageIQ/manageiq-api-client-python ]
+author: Daniel Korn (@dkorn)
+options:
+  miq_url:
+    description:
+      - the manageiq environment url
+    default: MIQ_URL env var if set. otherwise, it is required to pass it
+  miq_username:
+    description:
+      - manageiq username
+    default: MIQ_USERNAME env var if set. otherwise, it is required to pass it
+  miq_password:
+    description:
+      - manageiq password
+    default: MIQ_PASSWORD env var if set. otherwise, it is required to pass it
+  userid:
+    description:
+      - the unique userid in manageiq, often mentioned as username
+    required: true
+    default: null
+  username:
+    description:
+      - the users' full name
```

Comment:
On second thoughts, given the context of a "manageiq_user" module, consider `userid` being just `name`.
(Of course assuming the other param is renamed to something like `fullname`)

Many ansible modules use just "name" to identify the thing they operate on; specifically
https://docs.ansible.com/ansible/user_module.html,
https://docs.ansible.com/ansible/win_user_module.html,
https://docs.ansible.com/ansible/postgresql_user_module.html,
https://docs.ansible.com/ansible/os_user_module.html
all use `name` — "Name of the user to create, remove or modify."

@kwoodson what do you think?